README.md

```diff
@@ -14,12 +18,20 @@ All available templates (`expr` means the expression on which the template is ap
 | **.else**         | `if !expr` |
 | **.nil**         | `if expr == nil` |
 | **.notnil**      | `if expr != nil` |
+| **.nilr**      | `if expr == nil  { return expr }` |
+| **.must**      | `if expr != nil  { return expr }` |
```

Comment:
How about returning `\${0}` instead?

Reply:
Yes I need to correct this. I was thinking to make this must check for the `errors` because this seems to  be covered in the `nnil`.